### ISSUE TITLE
Videos recorded in portrait orientation are exported in landscape

### ISSUE THREAD

**Issue by simonmaddox** [AverageMarcus/video-to-gif#1]
Input:
[IMG_4478.mov.zip](https://github.com/AverageMarcus/video-to-gif/files/2132135/IMG_4478.mov.zip)
(Github doesn't support uploading of movie files, so I had to zip it)

Github won't let me attach the output gif because it's too large, but here's a screenshot:
![screen shot 2018-06-25 at 08 37 56](https://user-images.githubusercontent.com/34273/41836585-150ac22c-7853-11e8-8837-a4735b1bd1b7.png)

The video itself is from an iPhone Live Photo (take a photo, and the iPhone will also record a video at the same time) if that makes any difference.

**Comment by AverageMarcus** [comment]
When it was processing was the video orientated the correct way?

**Comment by simonmaddox** [comment]
Yes:

![screen shot 2018-06-25 at 09 05 48](https://user-images.githubusercontent.com/34273/41837980-fa12139a-7856-11e8-8d9b-cd8859625b24.png)

**Comment by AverageMarcus** [comment]
Oh that's interesting.

I'll look into it but not sure when I'll get the chance. I have a hunch that the iPhone is trying to do something clever with the video that gif.js isn't handling. 

Could you please also try with just a standard video to see if it's something special about the live photo?

**Comment by simonmaddox** [comment]
Oh no problem at all - I thought it best someone technical report it, rather than you receiving an email from a user saying "it doesn't convert correctly" without being able to provide any extra details.

Interestingly it works perfectly fine on a normal portrait orientation video, so I'd guess that there's some odd orientation metadata in the Live Photo version of a video.

**Comment by AverageMarcus** [comment]
Yeah that's what I thought. It might be something I have to raise with [gif.js](https://github.com/jnordberg/gif.js/) as I'm just making use of their library.
